packages/extension/src/providers/ethereum/libs/assets-handlers/assetinfo-mew.ts

```
@@ -243,15 +274,15 @@ export default (
-          supportedNetworks[networkName].cgPlatform as CoingeckoPlatform,
-        )
+        Object.keys(balances).filter(
+          contract => contract !== NATIVE_TOKEN_ADDRESS,
+        ),
+        supportedNetworks[networkName].cgPlatform as CoingeckoPlatform,
+      )
       : tokens.reduce(
-          (obj, cur) => ({ ...obj, [cur.contract]: null }),
-          {} as Record<string, CoinGeckoTokenMarket | null>,
-        );
+        (obj, cur) => ({ ...obj, [cur.contract]: null }),
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)